integration/config/orderer.yaml

```diff
@@ -147,6 +146,7 @@ General:
         # client's time as specified in a client request message
         TimeWindow: 15m
 
+
```

Comment:
extra  lines?

Reply:
this was extra space compared to the "template" from sampleconfig (i.e., when you do a `diff -r -u ../../hyperledger/fabric/sampleconfig integration/config`)